src/base/datetime/calendar/body/test/accessibility.test.ts

```diff
@@ -0,0 +1,230 @@
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(new KeyboardEvent("keydown", { key: "Right" }));
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.nextDay);
+    });
+
+    it('should be highlight next day when triggered "ArrowRight" keyboard event', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowRight" })
+      );
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.nextDay);
+    });
+
+    it('should changed value when pressing "Enter" key', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(
+        new KeyboardEvent("keydown", { key: "ArrowRight" })
+      );
+      await elementUpdated(el);
+
+      selectedEl.dispatchEvent(new KeyboardEvent("keydown", { key: "Enter" }));
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.nextDay);
+    });
+
+    it("should do nothing when pressing not handled key", async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = InitValue.month;
+      container.year = InitValue.year;
+      container.value = InitValue.value;
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
+        '.kuc-base-datetime-calendar-body__date[aria-selected="true"]'
+      )[0] as HTMLButtonElement;
+      selectedEl.click();
+      selectedEl.dispatchEvent(new KeyboardEvent("keydown", { key: "a" }));
+      await elementUpdated(el);
+      expect(container.value).to.equal(InitValue.value);
+    });
+
+    it('should be highlight next year when triggered "ArrowRight" keyboard event', async () => {
+      const container = new BaseDateTimeCalendarBody();
+      container.month = 11;
+      container.year = 2021;
+      container.value = "2021-12-31";
+
+      const el = await fixture(container);
+      const selectedEl = el.querySelectorAll(
```

Comment:
@nmanhit 
Please using el.querySelector() instead of el.querySelectorAll() to select first element.